pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -0,0 +1,65 @@
+package rest
+
+import (
+	"context"
+	"fmt"
+
+	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+type DualWriterMode1 struct {
+	DualWriter
+}
+
+// NewDualWriterMode1 returns a new DualWriter in mode 1.
+func NewDualWriterMode1(legacy LegacyStorage, storage Storage) *DualWriterMode1 {
+	return &DualWriterMode1{*newDualWriter(legacy, storage)}
+}
+
+func (d *DualWriterMode1) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
+	legacy, ok := d.legacy.(rest.Creater)
+	if !ok {
+		return nil, fmt.Errorf("legacy storage rest.Creater is missing")
```

Comment:
I think it would be better to use the [grafana's logging package](https://github.com/grafana/grafana/tree/main/pkg/infra/log) to keep logs consistent

Reply:
This was leading to some linting errors about importing Grafana core so I'm adding it to the ticket as a todo instead.